compiler/modules.nim

```diff
@@ -133,11 +133,12 @@ proc newModule(graph: ModuleGraph; fileIdx: int32): PSym =
     rawMessage(errInvalidModuleName, result.name.s)
 
   result.info = newLineInfo(fileIdx, 1, 1)
-  let pack = getIdent(getPackageName(filename))
+  let
+    pck = getPackageName(filename)
+    pck2 = if pck.len > 0: pck else: "unknown-package"
```

Comment:
why change this to 'unknown-package'? Please don't. These names might end up in the name hashes resulting in huge recompile times.